src/diffusers/loaders/ip_adapter.py

```diff
@@ -45,14 +46,15 @@ class IPAdapterMixin:
     @validate_hf_hub_args
     def load_ip_adapter(
         self,
-        pretrained_model_name_or_path_or_dict: Union[str, Dict[str, torch.Tensor]],
-        subfolder: str,
-        weight_name: str,
+        pretrained_model_name_or_path_or_dict: Union[str, List[str], Dict[str, torch.Tensor]],
+        subfolder: Union[str, List[str]],
+        weight_name: Union[str, List[str]],
         **kwargs,
     ):
         """
         Parameters:
-            pretrained_model_name_or_path_or_dict (`str` or `os.PathLike` or `dict`):
+            pretrained_model_name_or_path_or_dict (`str` or `
+            .PathLike` or `dict`):
```

Comment:
```suggestion
            pretrained_model_name_or_path_or_dict (`str` or `os.PathLike` or `dict`):
```

Reply:
But this is not the case no? We have `List[str]` as well as per the type-annoation.